src/redash_stmo/datasource_health/health.py

```diff
@@ -0,0 +1,92 @@
+import json
+import time
+import redash
+import settings
```

Comment:
No relative implicit imports please, use instead: `from . import settings` to make this code Python 3 compatible.